examples/c/wind_turbine_nmpc.c

```diff
@@ -697,6 +795,34 @@ int main()
 
 	ocp_nlp_solver *solver = ocp_nlp_create(config, dims, nlp_opts);
 
+	/************************************************
+	* 	precomputation (after all options are set)
+	************************************************/
+
+	for (int i=0; i<NN; i++){
+		if (plan->sim_solver_plan[i].sim_solver == GNSF)
```

Comment:
Did you try to play around with using GNSF on only a subset of the shooting intervals and make sure that works too?

Reply:
nope. But should not be a problem. I can try this apart from the pull request.

Reply:
I tried it out. It works fine